Components/Lists/Lists.js

```diff
@@ -2,354 +2,397 @@
+					<Text
+						style={[
+							styles.liveText,
+							{
+								fontFamily: 'cutive-mono-regular'
+							}
+						]}
+						accessibilityLabel={this.state.isRecording}>
+						
+						{this.state.isRecording ? 'RECORDING' : 'STOPPED RECORDING'}
+					</Text>
+				</View>
+			</View>
+		);
+	};
 
-getAllCaretakerLists = async () => {
-  const lists = await fetchCaretakerLists(this.state.caretaker_id);
-  this.props.loadLists(lists);
-    return lists.map(list => {
-    list = { ...list, client_id: user.id }
-    return (
-      <CaretakerList/>
-    );
-  }).reverse()
+	getClientLists = () => {
+		const { lists, user } = this.props;
+		return lists
+			.map(list => {
+				list = {
+					...list,
+					client_id: user.id
+				};
+				return <ClientList />;
+			})
+			.reverse();
+	};
+	//need to complete List component and pass props/methods
 
-} 
-//need to finish this function
+	getAllCaretakerLists = async () => {
+		const lists = await fetchCaretakerLists(this.state.caretaker_id);
+		this.props.loadLists(lists);
+		return lists
+			.map(list => {
+				list = {
+					...list,
+					client_id: user.id
+				};
+				return <CaretakerList />;
+			})
+			.reverse();
+	};
+	//need to finish this function
 
+	render () {
+		if (!this.state.fontLoaded) {
+			return <View style={styles.emptyContainer} />;
+		}
 
-  render() {
-    if(!this.state.fontLoaded) {
-      return (
-          <View style={styles.emptyContainer} />
-      )
-  }
+		if (!this.state.haveRecordingPermissions) {
+			return (
+				<View style={styles.container}>
+					<View />
+					<Text
+						style={[
+							styles.noPermissionsText,
+							{
+								fontFamily: 'cutive-mono-regular'
+							}
+						]}>
+						You must enable audio recording permissions in order to use this app.
+					</Text>
+					<View />
+				</View>
+			);
+		}
 
-  if (!this.state.haveRecordingPermissions){
-      return (
-          <View style={styles.container}>
-              <View />
-              <Text style={[styles.noPermissionsText, { fontFamily: 'cutive-mono-regular' }]}>
-                You must enable audio recording permissions in order to use this app.
-              </Text>
-              <View />
-          </View>
-      )
-  }
-    
-    return (
-      <View>
-      <View style={styles.headerContainer}>
-          <Text style={styles.header}>My Todo Lists</Text>
-        </View>
-         {this.props.user.accountType === "client" && this.recordListName()}
-         <ScrollView>
-         {this.props.user.accountType === "client" && this.getClientLists()}
-         {this.props.user.accountType === "caretaker" && this.getCaretakerLists()}
-         </ScrollView>
-      </View>
-    );
-  }
+		return (
+			<View>
+				<View style={styles.headerContainer}>
+					<Text style={styles.header}> My Todo Lists </Text>
+				</View>
+				{this.props.user.role=== 'client' && this.recordListName()}
+				<ScrollView>
+					{this.props.user.role=== 'client' && this.getClientLists()}
```

Comment:
I don't know if you got this warning, but when I tried to invoke a function as a component, the app told me that it's a violation. I couldn't view anything on the screen. If it worked for you, that's great. Just something to keep in mind.